### PR TITLE
Fix exception Call to a member function getRoute() on null

### PR DESCRIPTION
**Pull request by sadortun**
FatalThrowableError in /var/www/orocrm/vendor/oro/platform/src/Oro/Bundle/EmailBundle/Provider/EmailActivityListProvider.php line 473:
Call to a member function getRoute() on null

### CODE REVIEW

File: src/Oro/Bundle/EmailBundle/Provider/EmailActivityListProvider.php [hunk 1]
```diff
@@ -462,8 +462,13 @@ protected function setReplaedEmailId($email, $data)
      */
     protected function setOwnerLink($owner, $data)
     {
-        $route = $this->configManager->getEntityMetadata(ClassUtils::getClass($owner))
-            ->getRoute('view');
+        $route = null;
+        $entityMetadata = $this->configManager->getEntityMetadata(ClassUtils::getClass($owner));
+        
+        if(null !== $entityMetadata) {
```
Comment (br0ther) [inline src/Oro/Bundle/EmailBundle/Provider/EmailActivityListProvider.php]: CS: please add space after IF keyword

**Comment by br0ther** [comment]
Thank you for reporting the issue and the proposed bug fix.
Internal ID: BAP-15100

**Comment by anyt** [comment]
Fixed in https://github.com/oroinc/platform/commit/cca2ade3197023ed31ecddfd9fd6621625fd0451,
Thank you!

**Comment by sadortun** [comment]
Thanks

**Pull request closed without merging**